data/master.yaml

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
beautiful :smile_cat: 

maybe at a comment at the top: this file was generated ... do not edit manually bla bli blo